packages/grafana-ui/src/components/Forms/Input/Input.tsx

```diff
@@ -246,7 +245,7 @@ export const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
 
         {(suffix || loading) && (
           <div className={styles.suffix} ref={suffixRef}>
-            {loading && <Icon name="spinner" className={cx('fa-spin', styles.loadingIndicator)} />}
+            {loading && <i className={cx('fa fa-spinner', 'fa-spin', styles.loadingIndicator)} />}
```

Comment:
this looks strange :)

Reply:
:D Temporary fix,  as we don't have spinner icon.  Also I have converted Icons that have specific styling to <i /> to make sure that we are not losing any styling. I will definitely refactor it to <Icon /> when we agree on how </Icon > is implemented.